### ISSUE TITLE
Translate image alt texts

### ISSUE THREAD

**Issue by lukasrad02** [fsr-de/myHPI#390]
When adding an image with alt text to a page, the alt text does not show up in the translation view and therefore cannot be translated.

**Comment by abc013** [comment]
See https://github.com/wagtail/wagtail-localize/discussions/378: This is not yet supported by wagtail_localise and we would have to do it on our own.
